addons/mail/static/src/widgets/form_renderer/form_renderer.js

```diff
@@ -183,6 +200,17 @@ FormRenderer.include({
      * @param {mail.thread} ev.data.thread
      */
     _onChatterRendered(ev) {},
+    /**
+     * @private
+     * @param {MouseEvent} ev
+     */
+    _onScrollForm(ev) {
+        const target = ev.target;
+        const chatterContainer = target.querySelector('.o_FormRenderer_chatterContainer');
+        if (this._chatterContainerComponent && chatterContainer && !chatterContainer.classList.contains('o-aside')) {
```

Comment:
The form rendered itself is already aware if the chatter is aside or not (see `_isChatterAside`), this shouldn't check the DOM.

Note that this makes sense only in enterprise, chatter is never aside in community.